modlog_wiki_publisher.py

```
@@ -3,900 +3,1331 @@
+    }
+    
+    prefix = prefixes.get(target_type, 'ZZU')
+    
+    # Shorten long IDs for display
+    if len(str(target_id)) > 8 and target_type in ['post', 'comment']:
+        short_id = str(target_id)[:6]
+        return f"{prefix}{short_id}"
+    else:
+        return f"{prefix}{target_id}"
+
+def get_target_permalink(action):
```

Ruff (0.15.12)

[warning] Too many return statements (8 > 6)

(PLR0911)